### PR TITLE
Adding a migration command and safer icon renders

### PR DESCRIPTION
**Pull request by dvargasfuertes**
Adding a command palette command to help users with migrating from RoamJS sidebar

Also added safer checks for double icon render

### CODE REVIEW

File: src/sidebar.js [hunk 1]
```diff
@@ -71,7 +71,15 @@ const initializeRoamJSSidebarFeatures = (extensionAPI) => {
...
-      if (rightSidebarTopbar && extensionAPI.settings.get("ws-exp-col")) {
+      if (
+        rightSidebarTopbar &&
+        extensionAPI.settings.get("ws-exp-col") &&
+        !rightSidebarTopbar.hasAttribute("data-roamjs-sidebar-expcolall")
+      ) {
+        rightSidebarTopbar.setAttribute(
+          "data-roamjs-sidebar-expcolall",
+          "true"
+        );
```
Comment (dvargasfuertes) [inline src/sidebar.js:82]: I wasn't able to reproduce the config issue, but this should at least make it so that multiple icons don't appear

Comment (mlava) [reply src/sidebar.js:82]: Does that mean that the features weren't enabled onload while switched off in config? I'm definitely getting the buttons showing even when the switches are off. 

(I just saw where you refer to the config switches in sidebar.js, sorry for not seeing sooner)

**Review by mlava — APPROVED** [review]

File: src/sidebar.js (hunk 1 above)
Comment (dvargasfuertes) [reply src/sidebar.js:82]: That would be unexpected, we check to see if the switch is enabled right there in line 76

`extensionAPI.settings.get("ws-exp-col")`

When the switch is off but the icon is there, are you able to inspect what value this setting has that's returning truthy?

**Comment by mlava** [comment]
Yeah, it's weird right. I saw the switch check in sidebar.js. Maybe a weird leftover from previous testing? Let's hope the update will fix. 🙂

**Pull request merged**